python_functionality/homework.py

```diff
@@ -0,0 +1,128 @@
+from functools import reduce
```

Comment:
reduce not used. Comment or delete this line

Reply:
delete line